app/Http/Controllers/BotController.php

```diff
@@ -52,9 +58,15 @@ public function store(BotCreateRequest $request)
         try {
             $bot = $this->botRepository->create($data);
             return redirect()->route('bots.edit', $bot)
-                             ->with('messageSuccess', 'This bot successfully created');
+                ->with('messageSuccess', [
+                    'status' => 'Create success',
+                    'message' => 'This bot successfully created'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/BotController.php</br>**Line**: 63</br>Each line in an array declaration must end in a comma